src/SWP/Component/TemplatesSystem/Gimme/Event/MetaEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Templates System.
+ *
+ * Copyright 2019 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2019 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Component\TemplatesSystem\Gimme\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+final class MetaEvent extends Event
+{
+    private $isResultSet = false;
+
+    private $result;
+
+    private $data;
+
+    private $propertyName;
+
+    public function __construct($data, string $propertyName)
+    {
+        $this->data = $data;
+        $this->propertyName = $propertyName;
+    }
+
+    public function getData()
```

Comment:
return types are missing for some of the functions in this class

Reply:
data can be mixed (object, array) so no option to use return type for it.